client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less

```diff
@@ -24,6 +24,18 @@
       font-size: 14px !important;
     }
   }
+  
+  .bio-vo-text-button {
```

Comment:
Is this about the specific variable outline or a general Carbon customization?

Reply:
I don't think this is general carbon customization. Nevertheless, we have to see in `variable-outline` project.